### PR TITLE
readme 和 隐私政策更改

### PR DESCRIPTION
**Pull request by ArthurZhou**
readme 更新

移除了隐私政策中rvchub有关的内容

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-[![Discord](https://img.shields.io/discord/1140304794976792707?logo=discord&label=discord)](https://discord.gg/fCxmEyFgAd)
+[![Modrinth](https://img.shields.io/modrinth/dt/reden?style=flat-square&label=Modrinth)](https://modrinth.com/mod/reden)
+[![Discord Server](https://img.shields.io/discord/1140304794976792707?logo=discord&label=discord)](https://discord.gg/fCxmEyFgAd)
 [Official Website](https://redenmc.com)
 
 Redstone EDEN
 
 **English** | [简体中文](./README.zh-CN.md)
 
+## Before you start
```
Comment (zly2006) [inline README.md]: remove this, redenmc.com does not collect data from all user operations and we will send a privacy notice when the player launches reden the first time. on the other hand, i think this part should also contain the terms of service but it has not been complete yet

File: README.md [hunk 2]
```diff
@@ -16,19 +21,54 @@ For more information, please check [our wiki page](https://wiki.redenmc.com/Undo
...
 ## RVC
-> WIP
+> Work in progress, functions listed below are implemented
+
+![image](https://github.com/ArthurZhou/reden-is-what-we-made/assets/89689293/b2d5f6de-1860-496e-8980-b4a2cd3abe95)
+
+RVC(Redstone Version Control), provides change history and diff analysis for your machines based on git. Press `R+L` to open RVC screen.
+
+### Update, commit and push
+***Notice: DO NOT use hotkey `R+G` to login to Github [Issue #103](https://github.com/zly2006/reden-is-what-we-made/issues/103)***
```
Comment (zly2006) [inline README.md:29]: why to stress this, you ve said "functions listed below are implemented"

File: README.md [hunk 3]
```diff
@@ -16,19 +21,54 @@ For more information, please check [our wiki page](https://wiki.redenmc.com/Undo
...
+- `back` go back to the previous tick  ***Planning***
+
+
 ## RVC
-> WIP
+> Work in progress, functions listed below are implemented
+
+![image](https://github.com/ArthurZhou/reden-is-what-we-made/assets/89689293/b2d5f6de-1860-496e-8980-b4a2cd3abe95)
+
+RVC(Redstone Version Control), provides change history and diff analysis for your machines based on git. Press `R+L` to open RVC screen.
```
Comment (zly2006) [inline README.md]: change history -> version history

File: README.md [hunk 4]
```diff
@@ -16,19 +21,54 @@ For more information, please check [our wiki page](https://wiki.redenmc.com/Undo
...
-RVC, Redstone Version Control, machine history and diff analysis based on git.
+If your Reden account has been linked with GitHub(You can link it [here](https://redenmc.com/home/edit)), RVC can upload your machines to GitHub. 
 
-RVC can upload your machines to GitHub if you have bound your GitHub account at https://redenmc.com.
 RVC supports subregions and submodules, so you can use other people's work in your own creations.
 
+
+For the first-time use, you need to log in to your Reden account. After logging in, you will see the following:
+
+![image](https://github.com/ArthurZhou/reden-is-what-we-made/assets/89689293/f4bf3dee-9eca-4d0b-bada-82ced5fd6745)
```
Comment (zly2006) [inline README.md]: describe the image, dont use "image"

File: README.md [hunk 5]
```diff
@@ -1,13 +1,18 @@
 # Reden is What We Made
```
Comment (zly2006) [inline README.md]: use Reden plz

**Review by zly2006 — CHANGES_REQUESTED** [review]
also try gpt plz, it would help a lot

File: README.md (hunk 4 above)
Comment (zly2006) [reply README.md]: also, dont use hash as the filename, ** describe the image**

Comment (ArthurZhou) [reply README.md]: > also, dont use hash as the filename, ** describe the image**

github use hashes as url by default. I can't change that

File: README.md [hunk 6]
```diff
@@ -16,19 +17,54 @@ For more information, please check [our wiki page](https://wiki.redenmc.com/Undo
 All blocks, block entities and entities are undoable.
 Undo is available only if that change is caused by a player, for example, placing blocks and commands.
 
+
+## Control the game tick (tick)
```
Comment (zly2006) [inline README.md]: this command is carpet's

File: README.md [hunk 7]
```diff
@@ -16,19 +17,54 @@ For more information, please check [our wiki page](https://wiki.redenmc.com/Undo
...
+### Update, commit and push
+***Notice: DO NOT use hotkey `R+G` to login to Github [Issue #103](https://github.com/zly2006/reden-is-what-we-made/issues/103)***
+
+If your Reden account has been linked with GitHub(You can link it [here](https://redenmc.com/home/edit)), RVC can upload your machines to GitHub. 
 
-RVC can upload your machines to GitHub if you have bound your GitHub account at https://redenmc.com.
 RVC supports subregions and submodules, so you can use other people's work in your own creations.
 
+
+For the first-time use, you need to log in to your Reden account. After logging in, you will see the following:
```
Comment (zly2006) [inline README.md]: wtf is this, for online Minecraft players they can login automatically and wont see this gui

File: README.md [hunk 8]
```diff
@@ -16,19 +17,54 @@ For more information, please check [our wiki page](https://wiki.redenmc.com/Undo
...
+![rvc_login_success](https://github.com/ArthurZhou/reden-is-what-we-made/assets/89689293/f4bf3dee-9eca-4d0b-bada-82ced5fd6745)
+
+
+Then, just work with it like any other git repos:
+
+![rvc_git_funcs](https://github.com/ArthurZhou/reden-is-what-we-made/assets/89689293/de014ed2-e7b9-44d8-b100-9ab9d54523c4)
+
+### Block selection
 RVC brings a totally new way of selecting blocks that is much easier.
+
```
Comment (zly2006) [inline README.md]: do not add paragraph, ok??

**Review by zly2006 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (zly2006) [reply README.md]: > > also, dont use hash as the filename, ** describe the image**
> 
> github use hashes as url by default. I can't change that

???????

**Comment by zly2006** [comment]
@Cubik65536 yeah it is about docs so it's your turn now xd.
dont let me review any PR made by someone who has never read our code.

**Review by Qubik65536 — CHANGES_REQUESTED** [review]
Change whatever @zly2006 told you to change first. Use file name, you SHOULD NOT use hash for a file. And add **meaningful** alt for images.

File: README.md (hunk 4 above)
Comment (zly2006) [reply README.md]: > > also, dont use hash as the filename, ** describe the image**
> 
> github use hashes as url by default. I can't change that

oh dont drop files into the input box, you should upload the files via git to somewhere such as `docs/img` and then use them in markdown with a relative path

**Comment by ArthurZhou** [comment]
> Change whatever @zly2006 told you to change first. Use file name, you SHOULD NOT use hash for a file. And add **meaningful** alt for images.

github use hashes as file names when i upload images. i cant hack that

i've already add some alts for the images

**Comment by zly2006** [comment]
> > Change whatever @zly2006 told you to change first. Use file name, you SHOULD NOT use hash for a file. And add **meaningful** alt for images.
> 
> github use hashes as file names when i upload images. i cant hack that
> 
> i've already add some alts for the images

well you should learn to use your computer first before you start learning programming.

**Comment by ArthurZhou** [comment]
This doesn't seem to be a regular way 🤔

If you want to be the one who fuck the regulations, write a contribution guide to tell your contributors instead of losing your temper

**Comment by ArthurZhou** [comment]
And, btw, you can recognize images through alts, instead of hashes🤣

**Comment by Qubik65536** [comment]
> This doesn't seem to be a regular way 🤔
> 
> If you want to be the one who fuck the regulations, write a contribution guide to tell your contributors instead of losing your temper

You better respect the project maintainers and know how to add images properly to git repos AND know how to use correct link reference to ANY FILE.

**Pull request closed without merging**